platform/linux/mod.rs

```diff
@@ -691,21 +709,19 @@ fn recv(fd: c_int, blocking_mode: BlockingMode)
             return Ok((main_data_buffer, channels, shared_memory_regions))
         }
 
-        // Reassemble fragments.
-        let mut leftover_fragments = Vec::new();
+        // Reassomble fragments.
```

Comment:
Whoops... Not sure how this slipped through. I usually reread my diffs thoroughly to prevent this kind of fail :-(